tslearn/matrix_profile.py

```diff
@@ -71,15 +71,21 @@ class MatrixProfile(TransformerMixin,
     Parameters
     ----------
     subsequence_length : int (default: 1)
-        Length of the subseries (also called window size) to be used for 
+        Length of the subseries (also called window size) to be used for
         subseries distance computations.
-    
+
+    algorithm : str (default: None)
```

Comment:
I wouldn't call that "algorithm" since it's rather an implementation switch than an algorithm one at the moment.

Reply:
Would something like `kind` or `method` a la [`np.sort`](https://numpy.org/doc/1.18/reference/generated/numpy.sort.html) be appropriate?

Reply:
Yep, or "implementation"